wellcomeml/datasets/download.py

```diff
@@ -0,0 +1,45 @@
+import os
+import tarfile
+
+import boto3
+from botocore import UNSIGNED
+from botocore.client import Config
+
+from wellcomeml.logger import logger
+
+DATA_DIR = os.path.expanduser("~/.cache/wellcomeml/data")
+DATA_DISPATCH = {
+    "hoc": {
+        "bucket": "datalabs-public",
+        "path": "datasets/hoc/hoc.tar",
+        "file_name": "hoc.tar"
+    }
+}
+
+def check_cache_and_download(dataset_name):
```

Comment:
handy function!

Reply:
**thought**: I wonder whether it'd be good to keep it really generic so that we could use it for other S3 downloads too? e.g. have arguments for DATA_DIR and DATA_DISPATCH  and allow the case where credentials are needed?

Reply:
Note that https://github.com/wellcometrust/WellcomeML/blob/master/wellcomeml/ml/bert_vectorizer.py#L99-L121, we already use it for downloading SciBERT models too, so we might consider generalising it in a later work.